src/domain/state.ts

```diff
@@ -996,12 +1023,39 @@ function updateAppStateForApprovedApplication(
   if (updatePart.state === 'CLOSED') {
     return transitionToClosed(currentApplication, updatedBy);
   }
+  if (updatePart.state === 'PAUSED') {
+    if (currentApplication.state !== 'APPROVED') {
+      throw new Error('Cannot pause an application in this state');
+    }
+    // right now only SYSTEM role will be able to pause applications
+    if (updatedBy.role !== DacoRole.SYSTEM) {
+      throw new Error('Not allowed');
+    }
+    return transitionToPaused(currentApplication, updatedBy, updatePart.pauseReason);
+  }
   if (currentApplication.sections.ethicsLetter.declaredAsRequired && updateDocs) {
     delete updatePart.sections?.ethicsLetter?.declaredAsRequired;
     updateEthics(updatePart, currentApplication, updateDocs);
   }
 }
 
+function updateAppStateForPausedApplication(
```

Comment:
Added this in this PR so I could transition PAUSED apps back to APPROVED for testing